tests/TenantDatabaseManagerTest.php

```diff
@@ -225,7 +225,26 @@
 });
 
 test('path used by sqlite manager can be customized', function () {
-    pest()->markTestIncomplete();
+    Event::listen(TenantCreated::class, JobPipeline::make([CreateDatabase::class])->send(function (TenantCreated $event) {
+        return $event->tenant;
+    })->toListener());
+
+    // Set custom path for SQLite file
+    SQLiteDatabaseManager::$path = $customPath = database_path('custom');
+
+    if (! is_dir($customPath)) {
+        // Create custom directory
+        mkdir($customPath);
+    }
+
+    $name = Str::random(8). '.sqlite';
+    Tenant::create([
+        'tenancy_db_name' => $name,
+        'tenancy_db_connection' => 'sqlite',
+    ]);
+
+    expect(file_exists( $customPath . '/' . $name))->toBeTrue();
+    unlink($customPath . '/' . $name); // clean up
```

Comment:
It's better to use a random file than to do clean-up at the end of the test. What if the test failed mid-way? The file wouldn't get cleaned up and there'd be issues with running the test the next time around.

Reply:
Got it. Refactored the test to random dir name.

Reply:
Prefix the random string with something so that all of the databases created in this test start with the same string. We follow that practice everywhere. It's better than having *just* random characters.

Reply:
Ah, I see. That's a good pattern. I'll keep an eye out next time.